components/daikin_s21/s21.cpp

```diff
@@ -354,10 +368,15 @@ bool DaikinS21::run_queries(std::vector<std::string> queries) {
 }
 
 void DaikinS21::update() {
-  std::vector<std::string> queries = {"F1", "F5", "RH", "RI", "Ra", "RL", "Rd"};
-  if (this->run_queries(queries) && !this->ready) {
-    ESP_LOGI(TAG, "Daikin S21 Ready");
-    this->ready = true;
+  std::vector<std::string> queries = {"F1", "F5", "F9", "Rd"};
```

Comment:
My units do not support F9, so the inclusion of F9 as required breaks most of the functionality. Making it optional, or some sort of graceful degradation is required instead.

Reply:
Thanks for testing - another S21's quirk to add to the list. I fixed it in the force-push.

I was thinking of adding some kind of failure tracker, like trying all queries first and store successful ones to a new array, but I think that this is the least intrusive way to fix compatibility. Improved logic can be rewritten later.